packages/web/src/components/Plan.tsx

```diff
@@ -189,6 +190,10 @@ export class Plan extends React.Component<{}, State> {
         this.setState({ events: removeEvent(this.state.events, id) });
     };
 
+    removeAllEvents = (): void => {
+        this.setState({ events: removeAllEvents(this.state.events) });
```

Comment:
Does this work if you just set events to an empty array?

```js
this.setState({ events: [] });
```

So this way you won't need to add / use an extra `removeAllEvents` function

Reply:
yeah that works